UE/Application/Ports/UserPort.cpp

```diff
@@ -64,7 +64,16 @@ void UserPort::showSmsListView(std::vector<Sms> smsVector){
     ids.clear();
     smsListView.clearSelectionList();
     for(auto sms: smsVector){
-        smsListView.addSelectionListItem(std::to_string(sms.phoneNumber), "");
+        if (sms.read == true){
+            smsListView.addSelectionListItem(std::to_string(sms.phoneNumber), "");
+        }
+        else{
+            std::string test = std::to_string(sms.phoneNumber);
+            test=test+" Unread";
+            smsListView.addSelectionListItem(test, "");
+
```

Comment:
Niepotrzebna spacja + weź pod uwagę, że w tej metodzie obsługujemy 2 typy wiadomości wysłane przez nas i wysłane do nas.  Te wysłane przez nas nie mają mieć unread. Jeśli nie są to jest ok.

Reply:
Wiadomosci wyslane nie mają unread poniewaz smsy wysylane mają domyslnie ustawione pole read na true.